src/app/components/dashboard/dashboard.component.ts

```diff
@@ -538,8 +538,11 @@ export class DashboardComponent implements OnInit {
 
   async createBalanceChart(income, execIncome) {
     execIncome = execIncome || []
+
+    const ticksDecimalPlaces = 3
+
     // @ts-ignore     ¯\_(ツ)_/¯
-    Highstock.stockChart('highcharts' + this.randomChartId, {
+    Highstock.chart('highcharts' + this.randomChartId, {
```

Comment:
As mentioned, without that change the top tick was always missing for whatever reason.
Please make sure that nothing else has changed by that.